assets/messages.yml

```diff
@@ -447,6 +447,31 @@ messages:
 
         If you haven't already, you might like to make use of the [*+search feature+*|https://bugs.mojang.com/issues/?jql=project=%project_id%] to see if the issue has already been mentioned.
 
+        %quick_links%
+      fillname:
+        - Enter parent issue ID
+  duplicate-wf:
+    - project:
+        - bds
+        - mc
+        - mcapi
+        - mcd
+        - mce
+        - mcl
+        - mcpe
+        - mctest
+        - realms
+        - web
+      name: Duplicate (wf)
+      shortcut: wfdup
+      message: |-
+        *Thank you for your report!*
+        We're tracking this issue as *%s%*, so this ticket is being resolved and linked as a *duplicate*.
+
+        That ticket has already been resolved as Wont Fix, which means this is considered a bug but won't be fixed. Please do not leave a comment on the linked ticket.
```

Comment:
Would it maybe also make sense to name an example why a bug will not be fixed? E.g. something like the following:
>[...] but won't be fixed. One reason for this can be that the impact of the bug does not justify the amount of work required to fix it. The description of that ticket or the comments might explain the rationale. Please do not leave a comment on the linked ticket.

Reply:
There are too many reasons IMO and it might confuse the users thinking is one of those listed

Reply:
added the secnod part